Resources/Prototypes/holidays.yml

```diff
@@ -0,0 +1,369 @@
+
+- type: holiday
+  id: BastilleDay
+  name: Bastille Day
+  beginDay: 14
+  beginMonth: July
+
+- type: holiday
+  id: FriendshipDay
+  name: Friendship Day
+  beginDay: 30
+  beginMonth: July
+
+- type: holiday
+  id: BeerDay
+  name: Beer Day
+  shouldCelebrate:
+    !type:WeekdayInMonth
+    weekday: Friday
+  beginMonth: August
+
+- type: holiday
+  id: TalkLikeAPirateDay
+  name: Talk-Like-a-Pirate Day
+  beginDay: 19
+  beginMonth: September
+
+- type: holiday
+  id: ProgrammersDay
+  name: Programmers' Day
+  shouldCelebrate:
+    !type:DayOfYear
+    dayOfYear: 256
+
+- type: holiday
+  id: BisexualPrideDay
+  name: Bisexual Pride Day
+  beginDay: 23
+  beginMonth: September
+
+- type: holiday
+  id: StupidQuestionsDay
+  name: Stupid-Questions Day
+  beginDay: 28
+  beginMonth: September
+
+- type: holiday
+  id: AnimalsDay
+  name: Animal's Day
+  beginDay: 4
+  beginMonth: October
+
+- type: holiday
+  id: SmilingDay
+  name: Smiling Day
+  beginDay: 7
+  beginMonth: October
+
+- type: holiday
+  id: LesbianDay
+  name: Lesbian Day
+  beginDay: 8
+  beginMonth: October
+
+- type: holiday
+  id: CanadianThanksgiving
+  name: Thanksgiving in Canada
+  beginWeek: 2
+  beginWeekday: Monday
+  beginMonth: October
+
+- type: holiday
+  id: BossDay
+  name: Boss' Day
+  beginDay: 16
+  beginMonth: October
+
+- type: holiday
+  id: Halloween
+  name: Halloween
+  beginDay: 31
+  beginMonth: October
+
+- type: holiday
+  id: VeganDay
+  name: Vegan Day
+  beginDay: 1
+  beginMonth: November
+
+- type: holiday
+  id: ArmisticeDay
+  name: Armistice Day
+  beginDay: 11
+  beginMonth: November
+
+- type: holiday
+  id: KindnessDay
+  name: Kindness Day
+  beginDay: 13
+  beginMonth: November
+
+- type: holiday
+  id: LifeDay
+  name: Life Day
+  beginDay: 17
+  beginMonth: November
+
+- type: holiday
+  id: FlowersDay
+  name: Flower's Day
+  beginDay: 19
+  beginMonth: November
+
+- type: holiday
+  id: TransgenderRemembranceDay
+  name: Transgender Day of Remembrance
+  beginDay: 20
+  beginMonth: November
+
+- type: holiday
+  id: SayingHelloDay
+  name: Saying Hello Day
+  beginDay: 21
+  beginMonth: November
+
+- type: holiday
+  id: Thanksgiving
+  name: Thanksgiving in the United States
+  beginWeek: 4
+  beginWeekday: Thursday
+  beginMonth: November
+
+- type: holiday
+  id: Sinterklaas
+  name: Sinterklaas
+  beginDay: 5
+  beginMonth: December
+
+- type: holiday
+  id: HumanRightsDay
+  name: Human-Rights Day
+  beginDay: 10
+  beginMonth: December
+
+- type: holiday
+  id: MonkeyDay
+  name: Monkey Day
+  beginDay: 14
+  beginMonth: December
+
+- type: holiday
+  id: MayanDoomsday
+  name: Mayan Doomsday Anniversary
+  beginDay: 21
+  beginMonth: December
+
+- type: holiday
+  id: Christmas
+  name: Christmas
+  beginDay: 24
+  endDay: 26
+  beginMonth: December
+
+- type: holiday
+  id: FestiveSeason
+  name: Festive Season
+  beginDay: 1
+  endDay: 31
+  beginMonth: December
+
+- type: holiday
+  id: BoxingDay
+  name: Boxing Day
```

Comment:
Only really celebrated in UK, falls under Christmas?

Reply:
don't you dare touch boxing day.

if you're having US Independence day, we're having boxing day.

Reply:
> 
> 
> don't you dare touch boxing day.
> 
> if you're having US Independence day, we're having boxing day.

Should we even have US Independence day?

Reply:
no.

but we're still having boxing day.

:3c

Reply:
Straya also has it.